src/Workspaces/Core/Portable/Extensions/OperationExtensions.cs

```diff
@@ -57,7 +58,7 @@ public static ValueUsageInfo GetValueUsageInfo(this IOperation operation)
                         //
                         return ValueUsageInfo.Write;
 
-                    case IRecursivePatternOperation _:
+                    case IOperation iop when iop.GetType().GetInterfaces().Any(i => i.Name == "IRecursivePatternOperation"):
```

Comment:
Thanks for updating this. Can you also file a tracking issue to revert this IDE change?

Reply:
Added a note to [https://github.com/dotnet/roslyn/issues/33175](https://github.com/dotnet/roslyn/issues/33175).

---
In reply to: [254423185](https://github.com/dotnet/roslyn/pull/33178#discussion_r254423185) [](ancestors = 254423185)